spec/Subject-spec.ts

```diff
@@ -483,7 +483,7 @@ describe('Subject', () => {
     it('should handle subject never emits', () => {
       const observable = hot('-').asObservable();
 
-      expectObservable(observable).toBe(<any>[]);
```

Comment:
Changed because it fails otherwise since now the implementation actually expects a string as input